specifyweb/frontend/js_src/lib/components/FormMeta/index.tsx

```diff
@@ -179,7 +179,9 @@ function MetaDialog({
                   action="update"
                   tableName={resource.specifyModel.name}
                 >
-                  <MergeRecord resource={resource} />
+                  {resource.specifyModel.name === 'Agent' && (
```

Comment:
Instead of hardcoding the table name here, can you instead export and use the `recordMergingTables` Set in `*/lib/components/Merging/index.tsx`?

https://github.com/specify/specify7/blob/3bbc8378565eb66e9bdfa432d75e8ca6fb798339/specifyweb/frontend/js_src/lib/components/Merging/index.tsx#L42

That way, we can just add tables to this set and merging functionality will be enabled without having to find every place a table name is hardcoded for record merging. 

See this line for how to accomplish the logic:
https://github.com/specify/specify7/blob/3bbc8378565eb66e9bdfa432d75e8ca6fb798339/specifyweb/frontend/js_src/lib/components/Merging/index.tsx#L84